firefox-ios/Client/Frontend/Onboarding/ViewControllers/OnboardingInstructionPopupViewController.swift

```diff
@@ -64,23 +64,14 @@ class OnboardingInstructionPopupViewController: UIViewController, Themeable {
         stack.spacing = UX.textStackViewSpacing
     }
 
-    private lazy var primaryButton: LegacyResizableButton = .build { button in
-        button.titleLabel?.font = DefaultDynamicFontHelper.preferredBoldFont(
-            withTextStyle: .callout,
-            size: UX.buttonFontSize)
-        button.layer.cornerRadius = UX.buttonCornerRadius
-        button.titleLabel?.textAlignment = .center
+    private lazy var primaryButton: PrimaryRoundedButton = .build { button in
+        // Removed the lines that set font, corner radius, text alignment, adjustsFontForContentSizeCategory, and content edge insets
         button.addTarget(self, action: #selector(self.primaryAction), for: .touchUpInside)
-        button.titleLabel?.adjustsFontForContentSizeCategory = true
         button.accessibilityIdentifier = "\(self.viewModel.a11yIdRoot).DefaultBrowserSettings.PrimaryButton"
```

Comment:
This line can be removed as the accessibility identifier is set with the `PrimaryRoundedButtonViewModel` in `updateLayout()`.